pkg/glusterfs/controllerserver.go

```diff
@@ -65,40 +65,31 @@ type ProvisionerConfig struct {
 
 // ParseCreateVolRequest parse incoming volume create request and fill
 // ProvisionerConfig.
-//nolint gocyclo
 func (cs *ControllerServer) ParseCreateVolRequest(req *csi.CreateVolumeRequest) (*ProvisionerConfig, error) {
 
 	var (
 		reqConf ProvisionerConfig
 		gdReq   api.VolCreateReq
-		err     error
```

Comment:
Wondering when this `err` was added to this struct!!

Reply:
err was introduced in https://github.com/gluster/gluster-csi-driver/commit/77697b86799b3c92a65ac27a60b30f5d0cdbcc62 
and grouping was done in https://github.com/gluster/gluster-csi-driver/commit/13d257c9381a7d1f1a677d1b217895570c7264fb

Reply:
@Madhu-1 I was talking about pushing that error to the struct! that looks strange and admit that we missed it in review.